src/System.Management.Automation/security/SecureStringHelper.cs

```diff
@@ -189,9 +194,14 @@ internal static SecureString Unprotect(string input)
 
             protectedData = ByteArrayFromString(input);
 
+#if UNIX
+            // DPAPI isn't supported in UNIX, so we just translate the byte-array back to a string
+            data = ByteArrayFromString(input);
```

Comment:
Use the `protectedData` from above to avoid another call to the `ByteArrayFromString`.

Reply:
```suggestion
            data = protectedData;
```